jf_agent/git/bitbucket_server.py

```diff
@@ -234,9 +236,11 @@ def _normalize_commit(commit, repo, strip_text_content, redact_names_and_urls):
 
 
 def get_default_branch_commits(
-    client, api_repos, strip_text_content, server_git_instance_info, redact_names_and_urls
+    client, api_repos, strip_text_content, server_git_instance_info, redact_names_and_urls, verbose,
 ):
     for i, api_repo in enumerate(api_repos, start=1):
+        if verbose:
+            print(f"Beginning download of commits for repo {api_repo}")
```

Comment:
For these to make it to the log file, I believe this should be `agent_logging.log_and_print`

Reply:
We don't want them in the log file! Verbose logging might grow to include stuff a customer would rather not send us by default (though I'm not sure it does yet)-- this is for them to send us to help debug.

Reply:
Got it, :thumbsup: